README.tfdoc.hcl

```diff
@@ -0,0 +1,533 @@
+            Number of consecutive health checks successes required before
+            considering an unhealthy target healthy.
+          END
+        }
+
+        attribute "interval" {
+          type        = number
+          description = <<-END
+            Approximate amount of time, in seconds, between health checks of an
+            individual target. Minimum value `5` seconds, Maximum value `300`
+            seconds. For lambda target groups, it needs to be greater as the
+            timeout of the underlying lambda. 
+          END
+        }
+
+        attribute "matcher" {
+          type        = string
+          description = <<-END
+            Response codes to use when checking for a healthy responses from a
+            target. You can specify multiple values (for example, `"200,202"`
+            for HTTP(s) or `"0,12"` for GRPC) or a range of values (for example,
+            `"200-299"` or `"0-99"`). Required for HTTP/HTTPS/GRPC ALB. Only
+            applies to Application Load Balancers (i.e., HTTP/HTTPS/GRPC) not
+            Network Load Balancers (i.e., TCP).
+          END
+        }
+
+        attribute "path" {
+          type        = string
+          description = <<-END
+            Destination for the health check request. Required for
+            HTTP/HTTPS ALB and HTTP NLB. Only applies to HTTP/HTTPS.
+          END
+        }
+        attribute "port" {
+          type        = string
+          description = <<-END
+            Port to use to connect with the target. Valid values are either
+            ports `1`-`65535`, or `traffic-port`.
+          END
+        }
+
+        attribute "protocol" {
+          type        = string
+          description = <<-END
+            Protocol to use to connect with the target. 
+          END
+        }
+
+        attribute "timeout" {
+          type        = number
+          description = <<-END
+            Amount of time, in seconds, during which no response means a failed
+            health check. For Application Load Balancers, the range is `2` to
+            `120` seconds, and the default is `5` seconds for the instance
+            target type and `30` seconds for the lambda target type. For
+            Network Load Balancers, you cannot set a custom value, and the
+            default is `10` seconds for TCP and HTTPS health checks and `6`
+            seconds for HTTP health checks.
+          END
+        }
+
+        attribute "unhealthy_threshold" {
+          type        = number
+          description = <<-END
+            Number of consecutive health check failures required before
+            considering the target unhealthy. For Network Load Balancers, this
+            value must be the same as the `healthy_threshold`. 
+          END
+        }
+      }
+
+      variable "stickiness" {
+        type        = object(stickiness)
+        description = <<-END
+          Stickiness configuration block. 
+        END
+
```

Comment:
`duration` attribute seems to be missing

Reply:
Do you mean the `cookie_duration` attribute?

Reply:
sorry I waas wrong in this case!